Source/Fuse.Marshal/Marshal.Cast.uno

```diff
@@ -251,6 +251,7 @@ namespace Fuse
 			if (o is Size2) return (Size2) o;
 			else if (o is Size) return new Size2((Size)o, (Size)o);
 			else if (o is string) return StringToSize2((string)o);
+			else if (o is IArray) return ToSize2(ArrayToVector(o));
```

Comment:
I think it would be better to do `return ToSize2(ToVector((IArray)o));`, because the cast can't fail.